test/tests/utils/spline_interpolation/tests

```diff
@@ -9,6 +9,13 @@
 
     requirement = 'The system shall include a utility for computing spline functions.'
   []
+  [regular_spline_file]
+    type = 'Exodiff'
+    input = 'spline_interpolation_file.i'
+    exodiff = 'spline_interpolation_file_out.e'
+
+    requirement = 'The system shall include a utility for computing spline functions. The utility shall be able to read from a file or user input.'
+  []
```

Comment:
Can you do the following:

```
  [regular_spline]
    requirement = 'The system shall include a utility for computing spline functions that are input by'

    [param]
      type = 'Exodiff'
      input = 'spline_interpolation.i'
      exodiff = 'spline_interpolation_out.e'
      detail = 'parameters'
    []
    [file]
      type = 'Exodiff'
      input = 'spline_interpolation_file.i'
      exodiff = 'spline_interpolation_file_out.e'
      param = 'and by file'
    []
  []
```

And also add the issue to `issues`, i.e., 

```
[Tests]
  design = 'SplineFunction.md'
  issues = '#10918 #19298'
  ...
[]
```

Reply:
That's #19299, right? I can do that.

Reply:
The `param = 'and by file'` line gave an error. I couldn't find the documentation for the test harness HIT syntax, so I changed `param` to `detail`. If there was something else that `param` was supposed to do, let me know.

Reply:
Whoops - yeah that was just a mistype by me